Packs/CommonTypes/ReleaseNotes/3_2_42.md

```diff
@@ -0,0 +1,34 @@
+
+#### Layouts
+##### Email Indicator
+- Improved implementation of the layout.
```

Comment:
whats "implementation" mean?

Reply:
its a fixed prefix to our notes... its stupid but its the only one that made any sense...
```
'Added support for '
'Added the '
'Fixed an issue '
'Improved implementation '
'Updated the Docker image to '
'You can now '
'Deprecated. '
'Deprecated the'
```

Reply:
Maybe we should ask to add prefixes, looks like nobody took into account that we work on layouts, dashboards and playbooks which all have visual meanings in some way.

Reply:
can you open a dev task for what is missing please? Shahaf said they would add it

Reply:
@idovandijk - https://jira-hq.paloaltonetworks.local/browse/CIAC-3919